android/src/main/java/com/swmansion/rnscreens/gamma/tabs/TabsHost.kt

```diff
@@ -366,15 +373,19 @@ class TabsHost(
                     reactContext.assets,
                 )
 
-            val smallFontSize = tabBarItemTitleFontSize?.takeIf { it > 0 } ?: 12f
-            val largeFontSize = tabBarItemTitleFontSizeActive?.takeIf { it > 0 } ?: 14f
+            val smallFontSize =
+                tabBarItemTitleFontSize?.takeIf { it > 0 }?.let { it * resources.displayMetrics.density }
+                    ?: wrappedContext.resources.getDimension(com.google.android.material.R.dimen.design_bottom_navigation_text_size)
+            val largeFontSize =
+                tabBarItemTitleFontSizeActive?.takeIf { it > 0 }?.let { it * resources.displayMetrics.density }
+                    ?: wrappedContext.resources.getDimension(com.google.android.material.R.dimen.design_bottom_navigation_text_size)
 
             // Inactive
-            smallLabel.textSize = smallFontSize
+            smallLabel.setTextSize(TypedValue.COMPLEX_UNIT_PX, smallFontSize)
```

Comment:
Hmm, I'm not so sure about using px here. I have not looked up the spec now, but at a first glance it would make little sense for design system to specify dimensions in `px`. I expect `dip` is what we want & I believe it is usually useD in RN to specify corner radiuses etc. 

Can I ask you to verify this in reference specs & link them if they really specify values in `px`?